src/Components.tsx

```diff
@@ -1,6 +1,43 @@
+import React from 'react';
 import {useCreateComponent} from './useCreateComponent';
+import {useAttachAttribute} from './utils/useAttachAttribute';
 import {useUpdateWithSetter} from './utils/useUpdateWithSetter';
 
+export const ConnectPayments = (): JSX.Element => {
+  const {wrapper} = useCreateComponent('payments');
+  return wrapper;
+};
+
+export const ConnectPayouts = (): JSX.Element => {
+  const {wrapper} = useCreateComponent('payouts');
+  return wrapper;
+};
+
+export const ConnectPaymentDetails = ({
+  payment,
+  onClose,
+  visible = undefined,
```

Comment:
Let's not add visible here! It isn't documented, and we want people to stop using it